### PR TITLE
Create avatar.html

### PR DESCRIPTION
**Pull request by aniketsingh13**

### CODE REVIEW

File: styles/avatar.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (shubhamkumar648) [inline styles/avatar.html:7]: add proper title name

**Review by shubhamkumar648 — COMMENTED** [review]
good to go

File: styles/avatar.html (hunk 1 above)
Comment (aniketsingh13) [reply styles/avatar.html:7]: ok sir

File: styles/avatar.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>UI library</title>
```
Comment (shubhamkumar648) [inline styles/avatar.html:7]: good

**Review by shubhamkumar648 — COMMENTED** [review]
yo

**Pull request merged**